lib/filesystem.py

```diff
@@ -1175,7 +1175,7 @@ def init(self):
       if (v_found == STORAGE_VERSION):
          ch.VERBOSE("found storage dir v%d: %s" % (STORAGE_VERSION, self.root))
          self.lock()
-      elif (v_found in {None, 6}):  # initialize/upgrade
+      elif (v_found is None):  # initialize/upgrade
```

Comment:
I'd like to leave the set. Right now we're in an odd situation where literally no old storage versions are upgradeable, but I expect that will change in the future. So the set notation will make the intent clearer. Also my comment is kind of bad. For example:

```suggestion
      elif (v_found in {None,}):  # None if initializing, or upgraded version
```